plugins/cilium-cni/main.go

```diff
@@ -3,7 +3,7 @@
 
 // Ensure build fails on versions of Go that are not supported by Cilium.
 // This build tag should be kept in sync with the version specified in go.mod.
-//go:build go1.18
+//go:build go1.19
```

Comment:
any reason to rename this file to main.go ?

Reply:
Sorry, I forgot to mention this in the commit message. It's so this bit of `update-go-version` works for `plugins/cilium-cni` as well:

https://github.com/cilium/cilium/blob/09a68d6a5f5db01f8b73debce181f79d798fb8f2/Makefile#L615-L621

All other binaries' `main` funcs are already in `main.go` files, but this was probably missed.

Reply:
oh right, all good :100: